packages/fuel-indexer-tests/contracts/fuel-indexer-test/src/main.sw

```diff
@@ -42,6 +42,14 @@ pub struct SimpleTupleStruct {
     data: (u32, u64, str[12]),
 }
 
+pub struct ExplicitQueryStruct {
+    id: u64
+}
```

Comment:
would these ids need to be unique, or not?

Reply:
That probably don't need to be unique IDs between the two structs for the query tests, but I felt that it was good hygiene to make them unique so that we don't run into collisions later down the line.